assignments/homework/pcosta/497HW1.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy import units as u
+from astropy.units import imperial
+from astropy import constants as const
+from astropy.time import Time
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import EarthLocation, AltAz
+from random import *
```

Comment:
It is generally recommended not to use `*` imports, as it will populate your global namespace and can likely cause conflicts. The recommendation is to import only the things that you use.


Use explicit imports instead, in this case `import random` would be enough and then in the code it will also become clear where each function is coming from.